index.html

```diff
@@ -23,68 +23,60 @@ <h2>Taking the Stress Out of Working Out</h2>
-                    
+        <div class="search-exercise-div flex sm:flex-col lg:flex-row w-5/6 mx-auto">
+            <!-- Buttons/Search -->
+            <div class="buttons-div flex flex-col sm:w-5/6 lg:w-1/3">
+                <div class="search flex flex-col mx-auto rounded my-2 py-2.5 w-5/6 lg:py-auto lg:mt-0">
                     <!-- Buttons -->
-                        <button type="submit" class="button w-1/2 mx-auto mt-1 text-black rounded" id="target-btn">Abs</button>
+                    <button type="submit" class="button w-1/2 mx-auto mt-1 text-black rounded"
+                        id="target-btn">Abs</button>
+                </div>
 
+                <div class="search flex flex-col mx-auto rounded my-2 py-2.5 w-5/6 lg:py-auto lg:mt-0">
                     <!-- Search -->
-                    <input name="musclegroup" id="muscle-group" type="text" autofocus="true"
-                        class="form-input w-2/3 mx-auto" placeholder="muscle group" />
+                    <input name="musclegroup" id="muscle-group" type="text" autofocus="true" class="form-input mx-auto"
+                        placeholder="muscle group" />
                     <button type="submit" class="button w-1/2 mx-auto mt-1 text-white rounded" id="search-btn">
                         Get Started
                     </button>
-                </section>
-
+                </div>
+            </div>
 
-                <!-- Excercises -->
-                <section class="results w-5/6 lg:w-2/3 h-full flex flex-col mx-auto lg:ml-2 rounded py-2.5">
-                    <div class="description w-4/5 text-white px-2 py-2 mx-auto">
-                        <h3 class="exercise-title underline">Exercise Title</h3>
-                        <p class="exercise-details">
-                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi
-                            temporibus praesentium repudiandae iusto explicabo dolore quos
-                            distinctio dolorem repellendus repellat dignissimos cum officia
-                            provident sint, doloremque doloribus voluptatum? Suscipit, ipsa!
-                        </p>
-                    </div>
-                    <div class="image-video mt-2">
-                        <img src="./assets/images/vidplace.png" alt="exercise video image" class="video-placeholder w-4/5 mx-auto" />
-                    </div>
+            <!-- Excercises -->
+            <div class="results h-full flex flex-col lg:ml-2 rounded py-2.5 mx-auto sm:w-5/6 lg:w-2/3">
+                <section class="description w-4/5 text-white px-2 py-2 mx-auto">
+                    <h3 class="exercise-title underline">Exercise List</h3>
+                    <ul class="exercise-list"></ul>
+                </section>
                 </section>
             </div>
-        </div>
     </main>
 
     <!-- Footer -->
-    <footer>
+    <footer class="h-20">
```

Comment:
wrt to the strange footer comment that you made. I saw that this class "h-20" was here and not on the other one. When I removed it, it was the same as the exercises.html

You can play around with that and see which style you like better